types/onfleet__node-onfleet/Resources/Administrators.d.ts

```diff
@@ -1,44 +1,44 @@
 import { OnfleetMetadata } from '../metadata';
 
 declare class Admin {
-  create(obj: Admin.CreateAdminProps): Promise<Admin.OnfleetAdmin>;
```

Comment:
Let's leave out the formating changes in this PR. We could introduce a style config for the entire project separately.

Reply:
😎 removed the prettier stuff.